### PR TITLE
ImageNet || ResNet-50 || TensorFlow v1.11 + Cloud TPU v2

### PR DESCRIPTION
**Pull request by frankchn**

### CODE REVIEW

File: ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "hardware": "GCP n1-standard-2, Cloud TPU",
+  "costPerHour": 4.595,
+  "timestamp": "2018-09-29",
+  "misc": {
+    "optimizer": "SGD with Momentum",
+    "momentum": 0.9,
+    "weightDecay": 0.0001,
+    "batchSize": 1024,
+    "additional arguments": "--use_fast_lr=True --data_dir_small=<resized ImageNet directory>",
+    "additional info": "We have re-implemented the fast.ai techniques detaield in their 2018-04-20 submission to DAWNBench with progressive image sizes and a more aggressive learning rate schedule.",
```
Comment (brettkoonce) [inline ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json]: minor sp: detaield --> detailed
cool stuff!

Comment (frankchn) [reply ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json]: Fixed. Thanks!

File: ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "costPerHour": 4.595,
+  "timestamp": "2018-09-29",
+  "misc": {
+    "optimizer": "SGD with Momentum",
+    "momentum": 0.9,
+    "weightDecay": 0.0001,
+    "batchSize": 1024,
+    "additional arguments": "--use_fast_lr=True --data_dir_small=<resized ImageNet directory>",
+    "additional info": "We have re-implemented the fast.ai techniques detailed in their 2018-04-20 submission to DAWNBench with progressive image sizes and a more aggressive learning rate schedule.",
+    "warmUp": "The learning rate starts at 0, then it increases linearly per iteration for the first 4 epochs",
```
Comment (codyaustun) [inline ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json:18]: Do you need the same `evalBatchSize` note (i.e., `--eval_batch_size=1000`) to reproduce this experiment? If so, please add.

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good but minor question on eval.

File: ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json (hunk 2 above)
Comment (frankchn) [reply ImageNet/train/google_resnet50_tpu_tensorflow111_fastai.json:18]: Good point. Added.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Thanks! Looks good

**Pull request merged**